### ISSUE TITLE
Game Crashes when trying to craft a Dilute Mana Pool

### ISSUE THREAD

**Issue by GustoniaEagle** [Vazkii/Botania#1015]
The game crashes as soon as you put all of the crafting components into the crafting grid, and so I can't even craft the dilute mana pool. Playing on a server, see http://pastebin.com/0ppBGuCh for crash log.

**Comment by TheWhiteWolves** [comment]
Have you replicated it with only botania installed?, if not try to otherwise it will be assumed that another mod is causing the issue to occur and you will need to try it will each mod individually to find the one causing it.

**Comment by GustoniaEagle** [comment]
Haven't had a chance to test just Botania, but the diluted mana pool appears as fire in my NEI menu. The normal mana pool and the Everlasting pool both render fine.

**Comment by GustoniaEagle** [comment]
Further testing shows that both the Dilute Mana Pool and the Mana Flash appear as fire in NEI. When either of these items are visible in NEI, the console gets spammed with "*[Client thread/INFO] [STDOUT]: [net.minecraft.crash.CrashReport:func_85057_a:320]: Negative index in crash report handler (0/27)*".

Additionally, this still occurs when only the following mods are installed:
 Forge version 10.13.3.1408
 Baubles version 1.0.1.10
 Botania version r1.6-190
 CodeChickenCore version 1.0.6.44
 NotEnoughItems version 1.0.4.107

**Comment by MrKunji** [comment]
You shouldn't worry about the Mana Flash, it renders as fire in both NEI an WAILA without being an actual error.

**Comment by GustoniaEagle** [comment]
But the Dilute Mana Pool?

**Comment by Vazkii** [comment]
Does this happen without NEI?

**Comment by GustoniaEagle** [comment]
Removing NEI still causes the issue. The dilute pool won't even render in the creative menu, it will just crash. Crafting the pool still crashes.
Tried removing CodeChickenCore, still crashes.
The only mods in my instance now are Baubles (1.0.1.10) and Botania (r1.6-190). Using Forge 10.13.3.1408.

**Comment by Vazkii** [comment]
Either way, what I'm doing with my render is completely normal, so I can only assume this is a bug in forge. Try using an older version and seeing if it works.

**Comment by GustoniaEagle** [comment]
I believe I have found the cause of this. It really is an odd one, and I have no idea why this is happening.

I used MultiMC to make the test instance, and the instance that crashed initially. When I chose the Forge version from the list, as you would normally do, it doesn't give you the actual version of Forge, instead, it gives you this: http://puu.sh/i5I2X/6526ae3471.png
Note: the version has "(Custom)" at the end of it.

So, I tried to do the same instance using the Curse Client. I used the same Forge version as I had on MultiMC, and it worked fine with no crash.

To conclude, the MultiMC devs are doing something dodgy to their Forge versions, causing this crash.

**Comment by peterix** [comment]
The version is ALWAYS custom.

**Comment by GustoniaEagle** [comment]
Well that's fun. Turns out that somehow the Botania jar corrupted (might have been when I downloaded it, who knows?) and so caused the crash.
As a result, when I tried to make the same instance using the Curse Client, I used the Curse Client to download the jar, and so used a different and clearly, non-corrupted, jar. This caused me to think that MultiMC was the cause.

Apologies to both of you. @Vazkii @peterix
